src/js/containers/validateData/ValidateDataContainer.jsx

```diff
@@ -200,9 +215,15 @@ class ValidateDataContainer extends React.Component {
 			warningMessage = <PublishedSubmissionWarningBanner />;
 		}
 
+		let gtasWarning = null;
+		if(this.state.gtas && this.state.gtas.open){
```

Comment:
gtas.open no longer exists. Given the suggestion below you can probably just do the first part of this if for checking.